src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -27,12 +27,21 @@
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
+import java.net.HttpURLConnection;
 import java.net.InetAddress;
 import java.net.Socket;
+import java.net.URI;
+
```

Comment:
@cyberone Checkstyle will complain about this extra space, and I think import order is wrong (should be alphabetic).